docs/team/contributors-jupyterhub.yaml

```diff
@@ -32,3 +32,8 @@
   handle: "@willingc"
   affiliation: Project Jupyter
   contributions: Python,Community
+
+- name: Zach Sailer
+  handle: "@Zsailer"
+  affiliation: Project Jupyter
+  contributions: code,
```

Comment:
you're doing more than code, you're helping organize meetings! (happy to merge if you'd prefer to keep it to just code tho)

Reply:
Haha ok 😃. Do you think it's fair to say "code", "ideas/planning", and "question"? That covers the organizing and the helping with Github issues.  

Thanks for you review, Chris!